website/src/Pages/EventsPage/AddEventPage.jsx

```diff
@@ -0,0 +1,256 @@
+import { React, useState } from 'react';
+import {
+  Box, Button, Container, FormControl, Grid, MenuItem, Select, TextField, Typography,
+} from '@mui/material';
+import { useNavigate } from 'react-router-dom';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DatePicker, TimePicker } from '@mui/x-date-pickers';
+import dayjs from 'dayjs';
+import { useSelector } from 'react-redux';
+import axios from 'axios';
+import EventImageCard from './EventImageCard';
+
+function AddEventPage() {
+  const { authHeader } = useSelector((state) => state.sliceAuth);
+
+  const navigate = useNavigate();
+
+  const [title, setTitle] = useState('');
+  const [host, setHost] = useState('');
+  const [tag, setTag] = useState('');
+  const [date, setDate] = useState(dayjs());
+  const [recurrence, setRecurrence] = useState('');
+  const [startTime, setStartTime] = useState(dayjs());
+  const [endTime, setEndTime] = useState(dayjs().add(1, 'h'));
+  const [latitude, setLatitude] = useState('');
+  const [longitude, setLongitude] = useState('');
+  const [address, setAddress] = useState('');
+  const [description, setDescription] = useState('');
+  const [imageData, setImageData] = useState([]);
+  const [imagePreviews, setImagePreviews] = useState([]);
+
+  const backToEvents = () => {
+    navigate('/events');
+  };
+
+  const createEvent = async () => {
```

Comment:
validation might be a good idea for some of these fields, with popups for bad fields (i.e., non-numeric lat/long)

Reply:
(made this after starting Resource PR)

Reply:
also EditEvents page too

Reply:
I think we can keep this for the styling edit? @zhangjuliet let me know if you want me to implement this